features/main/url_encoded_id.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Allowing resource identifiers with characters that should be URL encoded
+  In order to have a resource with an id with special characters
+  As a client software developer
+  I need to be able to set and retrieve these resources with the URL encoded ID
+
+  @createSchema
+  Scenario Outline: Get a resource whether or not the id is URL encoded
+    Given there is a UrlEncodedId resource
+    And I add "Content-Type" header equal to "application/ld+json"
+    When I send a "GET" request to "<url>"
```

Comment:
shouldn't this be `/url_encoded_ids/%encode:id` I don't really understand how `<url>` matches the entity

Reply:
It's from the scenario outline variable below. Line 21. So I'm testing many different ways of fetching the entity with and without encoding.

Reply:
oh thanks I wasn't aware it was possible :)